src/Stratis.Bitcoin.Features.Miner/Controllers/MinerController.cs

```diff
@@ -114,5 +113,32 @@ public IActionResult StartStaking([FromBody]StartStakingRequest request)
                 return ErrorHelpers.BuildErrorResponse(HttpStatusCode.BadRequest, e.Message, e.ToString());
             }
         }
+
+        /// <summary>
+        /// Stop staking.
+        /// </summary>
+        /// <returns>An OKResult object that produces a status code 200 HTTP response.</returns>
```

Comment:
OKResult?

Reply:
That's the name of the type, I can add a ref=